README.md

```diff
@@ -35,7 +35,7 @@ The data formats, examples, and available web service interfaces will all change
 
 License
 -------
-By accessing, downloading, or using the APIs defined here you are agreeing to these terms of use [here.](https://github.com/spidasoftware/schema/raw/master/SPIDAWEB_EULA%20(19%20June%202019).pdf)
```

Comment:
we should probably delete the old eula pdf

Reply:
I wasn't able to find any pdf in schema